jepsen/src/jepsen/control/util.clj

```diff
@@ -80,13 +80,15 @@
   "Downloads a string URL and returns the filename as a string. Skips if the
   file already exists."
   ([url]
-   (wget! url false))
-  ([url force?]
+   (wget! url std-wget-opts false))
+  ([url opts]
+   (wget! url opts false))
+  ([url opts force?]
```

Comment:
Let's make `:force?` an option in the option map here, rather than having a separate argument for it.

Reply:
Is it a problem that this would break existing usages of `wget` with the force flag?

Reply:
Yeah, I'd check via `(map? opts)` to see whether it's a map being passed in, and dispatch appropriately.

Reply:
Also, these options are actually what get passed to `wget`. I realize now it's poorly named, so I will rename it to `wget-opts`. This way, `cached-wget` can pass in the `std-wget-opts` plus the additional flags `--user --pw`